app/views/messages/_message.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<% if current_user.id == message.sender_id %>
```

Comment:
I would mabe add from? to message so that you could write
`<% if message.from? current_user %>`